backend.py

```diff
@@ -803,6 +803,7 @@ def createDom0DiskFilesystems(install_type, disk, target_boot_mode, boot_partnum
     logs_partition = tool.getPartition(logs_partnum)
     if logs_partition:
         run_mkfs = True
+        change_logs_fs_type = diskutil.fs_type_from_device(partitionDevice(disk, logs_partnum)) != fs_type
```

Comment:
Why `change_logs_fs_type` and not `logs_fs_type` ?

Reply:
This is a boolean value to determine if we need to change the fs_type of the logs partition, hence the name
The end of the line has the comparison against the specified fs_type